Library/Homebrew/dev-cmd/contributions.rb

```diff
@@ -64,14 +64,15 @@ def contributions
         return ofail "Unsupported repository: #{repo}. Try one of #{SUPPORTED_REPOS.join(", ")}."
       end
 
+      tap = Tap.fetch("Homebrew", repo)
```

Comment:
Alternative to the tap code:

```suggestion
      repo_full_name = if repo == "brew"
        "Homebrew/brew"
      else
        tap = Tap.fetch("Homebrew", repo)
        tap.full_name
      end
```

and then change `tap.full_name` further down to be `repo_full_name`

Reply:
I’m sure I had this at one point, then I went down the “why is brew different, surely this is fairly common” question path and ended up in `tap.rb`